tools/scoring/score/dimensions/dimension.py

```diff
@@ -337,14 +337,14 @@ def match_virtual_entities(
         none_subscore_reference = _VirtualEntityMatch(
             correct=0,
             correct_ceiling=len(solution_raw_field_names),
```

Comment:
correct_ceiling and incorrect have the same values?

Reply:
why not binding them together?
`incorrect = MAX_CORRECT - correct`?

Reply:
can you please add a todo to bind them together

Reply:
I don't understand what you're asking: to reiterate our conversation on Friday, they are only the same value in this case. They are not inherently related. (See `:307` for another instance of `_VirtualEntityMatch` where the values differ.) Would this help?
```
correct_ceiling =len(solution_raw_field_names)
        none_subscore_reference = _VirtualEntityMatch(
            correct=0,
            correct_ceiling=correct_ceiling,
            incorrect=correct_ceiling,
```
Otherwise, a todo exists to pare down the attributes: https://trello.com/c/MBAeYiwI/31-pare-down-virtualentitymatch-attributes-following-implementation-of-entity-type-id